src/repr/src/scalar.rs

```diff
@@ -729,7 +729,11 @@ impl<'a> From<Numeric> for Datum<'a> {
 
 impl<'a> From<chrono::Duration> for Datum<'a> {
     fn from(duration: chrono::Duration) -> Datum<'a> {
-        Datum::Interval(Interval::new(0, 0, duration.num_microseconds().unwrap_or(0)).unwrap())
+        Datum::Interval(Interval::new(
+            0,
+            0,
+            duration.num_microseconds().unwrap_or(0),
```

Comment:
This would also look nicer as two statements:
```rust
let micros = duration.num_microseconds().unwrap_or(0);
Datum::Interval(Interval::new(0, 0, micros))
```